source/includes/k8s/deploy-operator.rst

```diff
@@ -14,43 +14,62 @@ Deploy the MinIO Operator
 Overview
 --------
 
-MinIO is a Kubernetes-native high performance object store with an S3-compatible
-API. The MinIO Kubernetes Operator supports deploying MinIO Tenants onto private
-and public cloud infrastructures ("Hybrid" Cloud).
-
-The following procedure installs the latest stable version
-(|operator-version-stable|) of the MinIO Operator and MinIO Plugin on Kubernetes
-infrastructure:
-
-- The MinIO Operator installs a :kube-docs:`Custom Resource Document (CRD)
-  <concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions>`
-  to support describing MinIO tenants as a Kubernetes :kube-docs:`object
-  <concepts/overview/working-with-objects/kubernetes-objects/>`. See the MinIO
-  Operator :minio-git:`CRD Reference <operator/blob/master/docs/tenant_crd.adoc>` for
-  complete documentation on the MinIO CRD.
-
-- The MinIO Kubernetes Plugin brings native support for deploying and managing
-  MinIO tenants on a Kubernetes cluster using the :mc:`kubectl minio` command. 
-
-This documentation assumes familiarity with all referenced Kubernetes
-concepts, utilities, and procedures. While this documentation *may* 
-provide guidance for configuring or deploying Kubernetes-related resources 
-on a best-effort basis, it is not a replacement for the official
-:kube-docs:`Kubernetes Documentation <>`.
+MinIO is a Kubernetes-native high performance object store with an S3-compatible API. 
+The MinIO Kubernetes Operator supports deploying MinIO Tenants onto private and public cloud infrastructures ("Hybrid" Cloud).
+
+The following procedure installs the latest stable version (|operator-version-stable|) of the MinIO Operator and MinIO Plugin on Kubernetes infrastructure:
+
+- The MinIO Operator installs a :kube-docs:`Custom Resource Document (CRD) <concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions>` to support describing MinIO tenants as a Kubernetes :kube-docs:`object <concepts/overview/working-with-objects/kubernetes-objects/>`. 
+  See the MinIO Operator :minio-git:`CRD Reference <operator/blob/master/docs/tenant_crd.adoc>` for complete documentation on the MinIO CRD.
+
+- The MinIO Kubernetes Plugin brings native support for deploying and managing MinIO tenants on a Kubernetes cluster using the :mc:`kubectl minio` command. 
+
+This documentation assumes familiarity with all referenced Kubernetes concepts, utilities, and procedures. 
```

Comment:
Style idea, perhaps drop `all` here? It's probably just me, but makes me wonder if there's going to be a test. (Even if that is extremely unlikely.)

Reply:
💡 Good idea.